### PR TITLE
Bake `docker-run.sh` into base clojure image

### PR DESCRIPTION
**Pull request by fraburnham**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,3 +16,5 @@ RUN curl -s -O http://www.ftp.cpc.ncep.noaa.gov/wd51we/wgrib2/wgrib2.tgz \
 RUN apt-get purge \
     && apt-get autoremove \
     && rm -rf /tmp/*
+
+COPY ./docker-run.sh /app/
```
Comment (whalesalad) [inline Dockerfile]: I might copy this into a more global `bin` directory.

Comment (fraburnham) [inline Dockerfile]: updated

**Comment by fraburnham** [comment]
The `Dockerfile` that is `FROM` this container can set `WORKDIR` to ensure `docker-run.sh` is called from the correct path.

Github maintains the 755 permissions for `docker-run.sh`

**Comment by whalesalad** [comment]
Go for it. +1

**Pull request merged**